mne_bids/tests/test_write.py

```diff
@@ -894,3 +894,64 @@ def test_write_anat(_bids_validate):
         write_anat(output_path, subject_id, t1w_mgh, session_id, raw=raw,
                    trans=trans, verbose=True, deface=dict(theta=100),
                    overwrite=True)
+    # Write some MRI data and supply `mri_landmarks`
+    mri_landmarks = [{'kind': 1, 'ident': 1,
+                      'r': np.array([66.085805, 51.33362, 46.52982013]),
+                      'coord_frame': FIFF.FIFFV_COORD_MRI},
+                     {'kind': 1, 'ident': 2,
+                      'r': np.array([41.873635, 32.246944, 74.553143]),
+                      'coord_frame': FIFF.FIFFV_COORD_MRI},
+                     {'kind': 1, 'ident': 3,
+                      'r': np.array([17.238125, 53.0829408, 47.017890]),
+                      'coord_frame': FIFF.FIFFV_COORD_MRI}]
+    mne.io.write_fiducials('mri_landmarks.fif', mri_landmarks,
+                           FIFF.FIFFV_COORD_MRI)
```

Comment:
can you use make_dig_montage instead and allow to pass a dig_montage to the landmarks parameters?
for me if you pass the landmarks this way it should bypass the use of trans. So trans must be None if you pass landmarks in MRI coordinates. Trans is only needed if you pass landmarks in "head" but I don't know if it's a real usecase. But It seems it's what you do below

Reply:
That works much better, great suggestion!